contracts/debt/IncurDebtV1.sol

```diff
@@ -0,0 +1,453 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import "../interfaces/IOHM.sol";
+import "../interfaces/IsOHM.sol";
+import "../interfaces/IgOHM.sol";
+import "../interfaces/IERC20.sol";
+import "../interfaces/IStaking.sol";
+import "./interfaces/ITreasury.sol";
+import "../libraries/SafeERC20.sol";
+import "../types/OlympusAccessControlledV2.sol";
+
+error IncurDebtV1_NotBorrower(address _borrower);
+error IncurDebtV1_InvaildNumber(uint256 _amount);
+error IncurDebtV1_WrongTokenAddress(address _token);
+error IncurDebtV1_AlreadyBorrower(address _borrower);
+error IncurDebtV1_AboveGlobalDebtLimit(uint256 _limit);
+error IncurDebtV1_AboveBorrowersDebtLimit(uint256 _limit);
+error IncurDebtV1_LimitBelowOutstandingDebt(uint256 _limit);
+error IncurDebtV1_AmountAboveBorrowerBalance(uint256 _amount);
+error IncurDebtV1_AmountMoreThanBorrowersLimit(uint256 _borrower);
```

Comment:
This is basically same as IncurDebtV1_AboveBorrowersDebtLimit() right. Can't we just use 1 exception for both of these.

Reply:
They are slightly different, but I guess we can.